x-pack/plugins/security_solution/server/lib/telemetry/helpers.ts

```diff
@@ -234,3 +238,31 @@ export const extractEndpointPolicyConfig = (policyData: PolicyData | null) => {
 export const addDefaultAdvancedPolicyConfigSettings = (policyConfig: PolicyConfig) => {
   return merge(DEFAULT_ADVANCED_POLICY_CONFIG_SETTINGS, policyConfig);
 };
+
+export const metricsResponseToValueListMetaData = ({
+  listMetricsResponse,
+  itemMetricsResponse,
+  exceptionListMetricsResponse,
+  indicatorMatchMetricsResponse,
+}: {
+  listMetricsResponse: ValueListResponseAggregation;
+  itemMetricsResponse: ValueListItemsResponseAggregation;
+  exceptionListMetricsResponse: ValueListExceptionListResponseAggregation;
+  indicatorMatchMetricsResponse: ValueListIndicatorMatchResponseAggregation;
+}) => ({
+  total_list_count: listMetricsResponse?.aggregations?.total_value_list_count ?? 0,
+  types:
+    listMetricsResponse?.aggregations?.type_breakdown?.buckets.map((breakdown) => ({
+      type: breakdown.key,
+      count: breakdown.doc_count,
+    })) ?? [],
+  lists:
+    itemMetricsResponse?.aggregations?.value_list_item_count?.buckets.map((itemCount) => ({
+      id: itemCount.key,
```

Comment:
Can we also include the type of each list here so we can associate the types with size information?